plugins/modules/zos_copy.py

```diff
@@ -449,15 +453,16 @@
     - VSAM data sets can only be copied to other VSAM data sets.
     - For supported character sets used to encode data, refer to the
       L(documentation,https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/resources/character_set.html).
-    - L(zos_copy,./zos_copy.html) uses SFTP (Secure File Transfer Protocol) for the underlying
-      transfer protocol; Co:Z SFTP is not supported. In the case of Co:z SFTP,
-      you can exempt the Ansible userid on z/OS from using Co:Z thus falling back
-      to using standard SFTP.
-    - Beginning in version 1.8.x, zos_copy will no longer attempt to autocorrect a copy of a data type member
-      into a PDSE that contains program objects. You can control this behavior using module option
-      executable that will signify an executable is being copied into a PDSE with other
-      executables. Mixing data type members with program objects will be responded with a
-      (FSUM8976,./zos_copy.html) error.
+    - This module uses SFTP (Secure File Transfer Protocol) for the underlying
+      transfer protocol; SCP (secure copy protocol) and Co:Z SFTP are not supported. In the
+      case of Co:z SFTP, you can exempt the Ansible user id on z/OS from using Co:Z thus falling
+      back to using standard SFTP. If the module detects SCP, it will temporarily use SFTP for
+      transfers, if not available, the module will fail.
+    - Beginning in version 1.8.x, zos_copy will no longer attempt to correct a copy of
+      a data type member into a PDSE that contains program objects. You can control this
+      behavior using module option executable that will signify an executable is being
+      copied into a PDSE with other executables. Mixing data type members with program
+      objects will be responded with a (FSUM8976,./zos_copy.html) error.
```

Comment:
Maybe throw a C() or I() around the module option above?
> ... option C(executable) that will signify ...

Not a deal-breaker, but since this section was already slightly re-worded, perhaps we could improve the last sentence:
> Mixing data type members with program objects will be responded with a (FSUM8976,./zos_copy.html) error.

perhaps something like:
> Mixing data type members with program objects will result in a (FSUM8976,./zos_copy.html) error.

Reply:
Done

Reply:
Looks like the last part of this sentence was changed for the 1.9.0 GA, but that update did not make it through to this version:
>Mixing data type members with program objects will result in a (FSUM8976,./zos_copy.html) error.
`responded with` --> `result`